### DISCUSSION TITLE
Firefly III is unable to format monetary amounts because your server is missing the required packages

### DISCUSSION THREAD

**Discussion by msy999** [firefly-iii/firefly-iii#8577] [General]
hi
I am Korean.
I don't speak English, so please understand.
  Firefly III is unable to format monetary amounts because your server is missing the required packages
The message always appears.

apt-get install -y language-pack-nl-base
locale-gen

There is no ko_KR.utf8 entry.

**Comment by JC5** [comment]
Try "language-pack-kr-base" and do a reboot.
